src/deploy/RPM_build/preparesrc.sh

```diff
@@ -1,36 +0,0 @@
-#!/bin/bash
```

Comment:
I think the reason this was separate initially so that we can have a different stage in Docker build - retries speed up in case of a temporary failure.